META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Main-Class: org.eln2.space.Space
+
```

Comment:
Not sure why you added this, explain?

Reply:
I didn't touch anything outside "integration-mc1-16"... Dunno what happened there...

Reply:
I would just delete the file then.